source/tests/system/nidigital_driver_api_tests.cpp

```diff
@@ -241,6 +271,54 @@ class NiDigitalDriverApiTest : public ::testing::Test {
   std::unique_ptr<::grpc::Server> server_;
 };
 
+TEST_F(NiDigitalDriverApiTest, PerformReadStatic_CompletesSuccessfully)
+{
+  const char* channel_name = "";
+  ::grpc::ClientContext context;
+  select_function(channel_name, digital::SelectedFunction::SELECTED_FUNCTION_NIDIGITAL_VAL_DIGITAL);
+  digital::ReadStaticRequest request;
+  request.mutable_vi()->set_id(GetSessionId());
+  request.set_channel_list(channel_name);
+  digital::ReadStaticResponse response;
+  ::grpc::Status status = GetStub()->ReadStatic(&context, request, &response);
```

Comment:
Consider checking other values on the response here like the `data` and `actual_num_read`. Same for the `frequencies` and `actual_num_frequencies` on the `FrequencyCounterMeasureFrequencyResponse` in the other test.

I believe we have these types of patterns tested through NI-Fake, though, so if the point of these tests is just to make sure we can make a successful API call then these tests seem fine as is.

Reply:
Yes, these type of patterns are checked in NI-Fake, that is why I did not include them here.